pkgs/applications/editors/mu/default.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, python3
+, fetchFromGitHub
+, qt5
+}:
+
+let
+  python = python3.override {
+    packageOverrides = self: super: {
+      pyqt5 = super.pyqt5.override { withSerialPort = true; };
+    };
+  };
+  python3Packages = python.pkgs;
```

```suggestion
  py = python3.override {
    packageOverrides = self: super: {
      pyqt5 = super.pyqt5.override { withSerialPort = true; };
    };
  };
```
We shouldn't alias actual variables that exist under pkgs.